### PR TITLE
Fix unit-tests for microbenchmarking

### PR DESCRIPTION
**Pull request by aleksbykov**
## PR pre-checks (self review)

- [ ] I followed [KISS principle](https://en.wikipedia.org/wiki/KISS_principle) and [best practices](https://docs.google.com/document/d/1jihgOKb5iGRlD8_HQ92O0JbLk1kASUoZT23i_MXFSKI)
- [ ] I gave variables/functions meaningful self-explanatory names
- [ ] I didn't leave commented-out/debugging code
- [ ] I didn't copy-paste code
- [ ] I added the relevant `backport` labels
- [ ] New configuration option are added and documented (in `sdcm/sct_config.py`)
- [ ] I have added tests to cover my changes (Infrastructure only - under `unit-test/` folder)
- [x] All new and existing unit tests passed (CI)
- [ ] I have updated the Readme/doc folder accordingly (if needed)

tests for creating  and verifying email report use check_regression method, which get data from ES. This could cause periodically that regression or enhancement blocks for expected results could appear or disappear from html report ( which could be expected). Added new verification of html report content, based on has_diff and has_improve flags

### CODE REVIEW

File: unit_tests/test_microbenchmarking.py [hunk 1]
```diff
@@ -167,9 +170,11 @@ def test_generate_html_report_for_avg_aio_with_zero_value(self):
         self.assertTrue(os.path.exists(report_file))
         self.assertGreater(os.path.getsize(report_file), 0)
         self.assertTrue(report_html)
-        self.assertIn('large-partition-forwarding', report_html)
-        # no regression for small-partition-slicing tests
-        self.assertNotIn('small-partition-slicing', report_html)
+        for key in report_results.keys():
```
Comment (bentsi) [inline unit_tests/test_microbenchmarking.py]: it looks like this is the same code from above (copy-paste)

**Comment by bentsi** [comment]
please explain in the details of the PR why this fixes the issue

**Comment by bentsi** [comment]
ping

File: unit_tests/test_microbenchmarking.py (hunk 1 above)
Comment (aleksbykov) [reply unit_tests/test_microbenchmarking.py]: Fixed. Tests are passed

**Comment by aleksbykov** [comment]
@bentsi, remove duplicate code, and adding explanation. Is it clear or should be rephrased?

**Comment by bentsi** [comment]
thanks Alex.

**Pull request merged**